jest.config.js

```diff
@@ -5,11 +5,13 @@ module.exports = {
     },
     moduleFileExtensions: ["js", "jsx", "ts", "tsx"],
     testEnvironment: "jsdom",
+    roots: ["<rootDir>/src/"],
     // From https://jestjs.io/docs/webpack#mocking-css-modules
     moduleNameMapper: {
         "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$":
             "<rootDir>/scripts/jestAssetTransformer.js",
         "\\.(css|less)$": "identity-obj-proxy",
     },
     transformIgnorePatterns: ["<rootDir>/node_modules/three/examples/(?!jsm/)"],
+    setupFiles: ["<rootDir>/src/test/testSetup.ts"],
```

Comment:
this is fixing problems with running the tests due to some missing dependencies that are present in the browser but not in the jsdom test environment